nsforest/plotting/_make_plots.py

```diff
@@ -64,37 +64,39 @@ def scatter_w_clusterSize(df, col, save = False, output_folder = "", outputfilen
         fig.write_html(filename)
     return fig 
 
-def dotplot(adata, markers, cluster_header, gene_symbols = None, dendrogram = True, save = False, output_folder = "", outputfilename_suffix = ""): 
+def dotplot(adata, markers, cluster_header, gene_symbols = None, dendrogram = True, save = False, show = False, output_folder = "", outputfilename_suffix = ""):
```

Comment:
The plots of interest use scanpy, which allows setting of save and show separately. Show is helpful when interactive, but problematic when running non-interactively, for example, in Nextflow. Allowing save to be optional just seems sensible and consistent.